### PR TITLE
First change in the Fork branch

### PR DESCRIPTION
**Pull request by Solano204**
Hay te va mi dog

### CODE REVIEW

**Comment by josue66656** [comment]
Okey

**Comment by Solano204** [comment]
WHATS UP MY DOG ??

**Review by josue66656 — COMMENTED** [review]
Okey tas pendejo

**Review by Solano204 — COMMENTED** [review]
why my dog

File: FACTORY/src/main/java/Entities/TypeEnemy.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package Proyecto.FACTORY.src.main.java.Entities;
```
Comment (josue66656) [inline FACTORY/src/main/java/Entities/TypeEnemy.java:1]: APUES MI PAQUETE PUTO

File: FACTORY/src/main/java/Main/Menu.java [hunk 2]
```diff
@@ -13,7 +14,7 @@ public static void main(String[] args) {
         
         // I just need to get any enemy child of the same interface and each one knows what to do (POLIMORFISM)
         // MY CHOICE WAS FUNG RED
-        Enemy enemySelected = FirstWay.getEnemySpecific(TypeEnemy.FUNGU_RED);
+        Enemy enemySelected = FirstWay.getEnemySpecific(TypeEnemy.ICE);
         enemySelected.attack();
```
Comment (josue66656) [inline FACTORY/src/main/java/Main/Menu.java:18]: nO ATACA

Comment (josue66656) [reply FACTORY/src/main/java/Main/Menu.java:18]: M

**Review by josue66656 — CHANGES_REQUESTED** [review]
JJJ

**Comment by josue66656** [comment]
JJNNN

**Pull request merged**